composer.json

```diff
@@ -11,22 +11,25 @@
             "email": "dcramer@gmail.com"
         }
     ],
-    "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2.1",
-        "phpunit/phpunit": "^4.8 || ^5.0",
-        "ext-openssl": "*",
-        "nokitakaze/testhttpserver": "^0.1.0",
-        "monolog/monolog": "~1.0"
-    },
     "require": {
-        "php": ">=5.5",
-        "ext-hash": "*",
+        "php": ">=5.6",
```

Comment:
Then we should also remove the nightly and hhvm(?) builds from Travis.

Or run `composer install --ignore-platform-reqs` for those builds (which seems like a good idea to be able tot test for 7.2 ahead of time).